go/stream/message.go

```diff
@@ -33,6 +34,7 @@ type MessageProperties struct {
 	SourceID        string    `json:"sourceID" validate:"required"`
 	ReceivedAt      time.Time `json:"receivedAt" validate:"required"`
 	RequestIP       string    `json:"requestIP" validate:"required"`
+	DestinationID   string    `json:"destinationID"`             // optional
```

Comment:
@achettyiitr We could have had `omitempty` here as well like for the other optional parameters, right?

Reply:
Sorry, missed this comment. Adding it in a separate PR.